independent-projects/resteasy-reactive/pom.xml

```diff
@@ -36,7 +36,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <!-- Versions -->
         <version.cdi>2.0.2</version.cdi>
-        <version.jandex>2.2.3.Final</version.jandex>
+        <version.jandex>2.2.4.Final-SNAPSHOT</version.jandex>
```

Comment:
I sure hope this goes away :)

Reply:
Jason says here https://github.com/wildfly/jandex/issues/84#issuecomment-765692608 that Jandex 3.0 will include this facility. (I should have mentioned this above, it's one more reason why this is a draft. When Jandex is released, this will obviously change, and the srcdeps stuff will go away.)